mantle/kola/tests/ostree/sync.go

```diff
@@ -0,0 +1,249 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ostree
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/coreos/coreos-assembler/mantle/kola"
+	"github.com/coreos/coreos-assembler/mantle/kola/cluster"
+	"github.com/coreos/coreos-assembler/mantle/kola/register"
+	"github.com/coreos/coreos-assembler/mantle/platform"
+	"github.com/coreos/coreos-assembler/mantle/platform/conf"
+	"github.com/coreos/coreos-assembler/mantle/platform/machine/qemu"
+	"github.com/coreos/coreos-assembler/mantle/util"
+)
+
+// https://github.com/coreos/coreos-assembler/pull/3998#issuecomment-2589994641
+var nfs_server_butane = conf.Butane(`variant: fcos
+version: 1.5.0
+storage:
+  directories:
+  - path: /var/nfs1/share
+    mode: 0777
+  - path: /var/nfs2/share
+    mode: 0777
+  - path: /var/nfs3/share
+    mode: 0777
+  - path: /var/nfs4/share
+    mode: 0777
+  - path: /var/nfs5/share
+    mode: 0777
+  - path: /var/nfs6/share
+    mode: 0777
+  files:
+    - path: "/etc/exports"
+      overwrite: true
+      contents:
+        inline: |
+          /var/nfs1/share  *(rw,insecure,no_root_squash)
+          /var/nfs2/share  *(rw,insecure,no_root_squash)
+          /var/nfs3/share  *(rw,insecure,no_root_squash)
+          /var/nfs4/share  *(rw,insecure,no_root_squash)
+          /var/nfs5/share  *(rw,insecure,no_root_squash)
+          /var/nfs6/share  *(rw,insecure,no_root_squash)
+    - path: "/var/lib/nfs/etab"
+systemd:
+  units:
+    - name: "nfs-server.service"
+      enabled: true`)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		// See https://github.com/ostreedev/ostree/pull/2968
+		Run:         ostreeSyncTest,
+		ClusterSize: 0,
+		Name:        "ostree.sync",
+		Description: "Verify ostree can sync the filesystem with disconnected the NFS volume.",
+		Distros:     []string{"rhcos"},
+		Platforms:   []string{"qemu"},
+		Tags:        []string{"ostree", kola.SkipBaseChecksTag, kola.NeedsInternetTag},
```

Comment:
Now that you don't use the container, is `NeedsInternetTag` still needed?

Reply:
I think we still need this, if removed, will fail with mount.
-- FAIL: ostree.sync (604.51s)
        harness.go:106: TIMEOUT[10m0s]: ssh: for i in $(seq 6); do
			sudo mount -t nfs4 10.0.2.2:/var/nfs$i/share /var/tmp/data$i
			done